openscm/scmdataframe/base.py

```diff
@@ -14,34 +14,27 @@
 import pandas as pd
 from dateutil import parser
 
-from openscm.core import Core
-from openscm.timeseries_converter import (
+from ..core.parameters import HIERARCHY_SEPARATOR
+from ..core.parameterset import ParameterSet
+from ..core.time import (
     ExtrapolationType,
     InterpolationType,
     ParameterType,
+    TimePoints,
     TimeseriesConverter,
 )
-from openscm.units import UnitConverter
-from openscm.utils import (
-    convert_datetime_to_openscm_time,
-    convert_openscm_time_to_datetime,
-    is_floatlike,
-)
-
+from ..core.units import UnitConverter
 from .filters import (
-    DEFAULT_SEPARATOR,
     datetime_match,
     day_match,
     hour_match,
-    is_str,
```

Comment:
can we do away with the `isinstance(s, six.string_types)` test because we only support Python3.7?

Reply:
yes